video.js

```diff
@@ -366,6 +367,17 @@ define(['./teletext', './utils'], function (Teletext, utils) {
         ////////////////////
         // Main drawing routine
         this.polltime = function (clocks) {
+            // Even with no light pen physically attached, the system VIA can
```

Comment:
It's a shame we have to do this work on every call to `polltime()`. It's the most-called routine in the emulator. That said: I can't see any way around it, unless you have a more cunning idea, so this is 100% ok :)